### PR TITLE
WIP A11y updates

### PR DESCRIPTION
**Pull request by jenweber**
**WIP: once this PR is approved, I'll do the work in the tutorials app. Once both repos have PRs, they can be merged**

Addresses #86. This PR reworks some of the changes made in https://github.com/ember-learn/super-rentals/pull/26

I have addressed the CSS errors. Todd, I reverted some of the work on roles because I couldn't resolve the warnings. I think certain role elements should be direct children/siblings and this app doesn't work that way right now. I'd rather show a simpler implementation that introduces a11y concerns but doesn't confuse new devs.

CSS changes are as slight as possible to possibly avoid redoing every single screenshot. Opinions on whether it's close enough?

cc @toddjordan @MelSumner

### CODE REVIEW

**Comment by jenweber** [comment]
My PR styling:
screen shot 2017-12-10 at 7 20 51 pm

Old PR styling:
screen shot 2017-12-10 at 7 22 37 pm

File: app/templates/components/rental-listing.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <article class="listing">
-  <a {{action 'toggleImageSize'}} class="image {{if isWide "wide"}}">
-    <img src="{{rental.image}}" alt="">
+  <a {{action 'toggleImageSize'}} class="image {{if isWide "wide"}}" aria-hidden="true" tabindex="-1">
```
Comment (MelSumner) [inline app/templates/components/rental-listing.hbs]: - Interactive elements are not supposed to be hidden- can you tell me what the purpose would be here? 
- Also, if you were to hide an element, you wouldn't need an alt attribute value on a child element.

**Review by MelSumner — DISMISSED** [review]

File: app/templates/components/rental-listing.hbs (hunk 1 above)
Comment (jenweber) [reply app/templates/components/rental-listing.hbs]: It was overlooked during copy & paste. I'll fix this and then get started on any edits to the tutorial

**Review by toddjordan — APPROVED** [review]

**Review by toddjordan — CHANGES_REQUESTED** [review]
Alright, a few of things before merging this one.  First, I resolved conflicts, but then had to fix a couple of things.  I pr'd those updates to your branch: https://github.com/jenweber/super-rentals/pull/1

Second, the changes you made to the css files need to go back into tutorial styles addon.  I see you made a pr so I will take care of that.

Third, we need to update the tutorial.  Its all you if you have bandwidth.  Otherwise I can take it on.

**Comment by toddjordan** [comment]
@jenweber The travis errors will go away once https://github.com/jenweber/super-rentals/pull/1 is merged

**Comment by toddjordan** [comment]
closing this one in favor of #121 (conflict resolutions, test fixes)

**Pull request closed without merging**